src/app/components/core/manage-services/list-manage-services.component.html

```diff
@@ -0,0 +1,99 @@
+                        <span class="icon">
+                          <i class="fas fa-ellipsis-v fa-xs fa-fw text-gray-600"></i>
+                        </span>
+                      </a>
+                    </div>
+                    <div class="dropdown-menu" id="dropdown-menu3" role="menu">
+                      <div class="dropdown-content">
+                        <a *ngIf="sharedService.checkAuth() && service?.status !== 'shutdown'" class="dropdown-item"
+                          (click)="setService(service);openModal('confirmation-dialog')">
+                          <span>Disable</span>
+                        </a>
+                        <a *ngIf="sharedService.checkAuth() && service?.status === 'shutdown'" class="dropdown-item"
+                          (click)="setService(service);openModal('confirmation-dialog')">
+                          <span>Enable</span>
+                        </a>
+                        <a *ngIf="sharedService.checkAuth()" class="dropdown-item"
+                          (click)="setService(service);openModal('delete-confirmation-dialog')">
+                          <span>Delete</span>
+                        </a>
+                      </div>
+                    </div>
+                  </div>
+                  <a *ngIf="!service?.name" (click)="openServiceModal(service?.type)" class="button is-text text-btn">
+                    {{service?.btntxt}}
+                  </a>
+                </td>
+              </tr>
+            </tbody>
+          </table>
+      </div>
+    </div>
+  </div>
+  <app-manage-service-modal (notify)="checkSelectedServiceStatus()" [serviceData]="serviceData"></app-manage-service-modal>
+  <app-confirmation-dialog id="delete-confirmation-dialog">
+    <header class="modal-card-head">
+      <p class="modal-card-title is-size-6">Delete</p>
+      <button type="button" class="delete" aria-label="close" (click)="closeModal('delete-confirmation-dialog')"></button>
+    </header>
+    <section class="modal-card-body">
+      Deleting this service can not be undone. Continue?
+    </section>
+    <footer class="modal-card-foot">
+      <button type="button" class="button is-small" (click)="closeModal('delete-confirmation-dialog')">Cancel</button>
+      <button type="button" class="button is-small is-danger" (click)="deleteService()" [appDisableUntilResponse]="reenableButton">Delete</button>
+    </footer>
+  </app-confirmation-dialog>
+  <app-confirmation-dialog id="confirmation-dialog">
+    <header class="modal-card-head">
+      <p class="modal-card-title is-size-6">{{service?.status === 'shutdown' ? 'Enable' : 'Disable'}}</p>
+      <button type="button" class="delete" aria-label="close" (click)="closeModal('confirmation-dialog')"></button>
+    </header>
+    <section class="modal-card-body">
+      Are you sure, You want to {{service?.status === 'shutdown' ? 'enable' : 'disable'}} the service {{service?.name}}?
```

Comment:
Have you tested a service a) added in disabled mode; or b) disabled & Fledge/FL restarted?

Reply:
yes